src/windows/player.c

```diff
@@ -304,6 +375,19 @@ void window_player_overview_mouse_down(int widgetIndex, rct_window *w, rct_widge
 	case WIDX_GROUP_DROPDOWN:
 		window_player_overview_show_group_dropdown(w, widget);
 		break;
+	case WIDX_KICK:
+		window_player_overview_show_kick_dropdown(w, widget);
+		break;
+	}
+}
+
+void window_player_overview_textinput(rct_window *w, int widgetIndex, char* text)
+{
+	if (!text) {
+		return;
+	}
+	if (widgetIndex == WIDX_KICK) {
+		window_player_overview_kick_player(w, STR_KICK_REASON_OTHER_REASON, _strdup(text));
```

Comment:
In this case you can just send `text` through. You are sending a pointer to the text, there is no need to copy the text anywhere in this function.